controllers/actions.github.com/resourcebuilder.go

```diff
@@ -545,14 +543,23 @@ func applyGitHubURLLabels(url string, labels map[string]string) error {
 	}
 
 	if len(githubConfig.Enterprise) > 0 {
-		labels[LabelKeyGitHubEnterprise] = githubConfig.Enterprise
+		labels[LabelKeyGitHubEnterprise] = trimLabelValue(githubConfig.Enterprise)
 	}
 	if len(githubConfig.Organization) > 0 {
-		labels[LabelKeyGitHubOrganization] = githubConfig.Organization
+		labels[LabelKeyGitHubOrganization] = trimLabelValue(githubConfig.Organization)
 	}
 	if len(githubConfig.Repository) > 0 {
-		labels[LabelKeyGitHubRepository] = githubConfig.Repository
+		labels[LabelKeyGitHubRepository] = trimLabelValue(githubConfig.Repository)
 	}
 
 	return nil
 }
+
+const trimLabelVauleSuffix = "-trim"
+
+func trimLabelValue(val string) string {
```

Comment:
do we have anything else need to trim? both in code and helm chart, might worth to double check all labels on an installed ARC.

Reply:
I'll test it :smile: